src/components/Menu/Menu.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Item } from "./Item/Item.jsx";
+import Pepperoni from "../../views/Pizzas/Pepperoni.jpg";
+import Slayer from "../../views/Pizzas/slayerpizza.jpg";
+import Alonepizza from "../../views/Pizzas/Alonepizza.jpg";
+import Kiwi from "../../views/Pizzas/kiwi.jpg";
+import Unicorn from "../../views/Pizzas/unicorn.jpg";
+import Potato from "../../views/Pizzas/Potato.jpg";
+
+const Menu = ({ name, desc }) => {
+  return (
+    <section className="features-boxed">
+      <div className="container">
+        <div className="intro">
+          <h2 className="text-center">{name}</h2>
+          <p className="text-center">{desc}</p>
+        </div>
+        <div className="row justify-content-center features">
+          <Item
```

Comment:
For now, keep the menu item's data in an array (of objects). You can create a separate `defaultMenuItems.js` file for it.

Create the whole menu by mapping through this array & creating an Item for every record, filled with this record's data.

Later on, we will be requesting menu data from the server, so we must be prepared for any number of dishes in the menu :)